link2aws.js

```diff
@@ -187,14 +190,17 @@ class ARN {
                 "workgroup": null,
             },
             "autoscaling": { // Amazon EC2 Auto Scaling
-                "autoScalingGroup": null,
+                "autoScalingGroup": () => {
+                    const groupName = this.resource.split('/')[1];
+                    return `https://${this.region}.console.aws.amazon.com/ec2/home?region=${this.region}#AutoScalingGroupDetails:id=${groupName};view=details`
```

Comment:
In all places where `console.aws.amazon.com` is hardcoded, can you use `${this.console}` instead? The console domain is different for AWS GovCloud and AWS China.